src/pymatgen/util/testing.py

```diff
@@ -18,42 +19,105 @@
 from monty.json import MontyDecoder, MontyEncoder, MSONable
 from monty.serialization import loadfn
 
-from pymatgen.core import ROOT, SETTINGS, Structure
+from pymatgen.core import ROOT, SETTINGS
 
 if TYPE_CHECKING:
     from collections.abc import Sequence
     from typing import Any, ClassVar
 
-MODULE_DIR = Path(__file__).absolute().parent
-STRUCTURES_DIR = MODULE_DIR / ".." / "structures"
-TEST_FILES_DIR = Path(SETTINGS.get("PMG_TEST_FILES_DIR", f"{ROOT}/../tests/files"))
-VASP_IN_DIR = f"{TEST_FILES_DIR}/io/vasp/inputs"
-VASP_OUT_DIR = f"{TEST_FILES_DIR}/io/vasp/outputs"
-# fake POTCARs have original header information, meaning properties like number of electrons,
+    from pymatgen.core import Structure
+    from pymatgen.util.typing import PathLike
+
+_MODULE_DIR: Path = Path(__file__).absolute().parent
```

Comment:
**Need confirm**: make `MODULE_DIR` private, as the [`util.testing` directory contains nothing else other than`__init__.py`](https://github.com/materialsproject/pymatgen/tree/master/src/pymatgen/util/testing), I guess it's used to define `STRUCTURES_DIR` only:

https://github.com/materialsproject/pymatgen/blob/31f1e1fb8cc1bb517d59ea8e484965bb641c42a0/src/pymatgen/util/testing/__init__.py#L27-L28